### DISCUSSION TITLE
Is it necessary to use iptables-legacy inplace of iptables?

### DISCUSSION THREAD

**Discussion by alfredsecure** [telekom-security/tpotce#1486] [Q&A] [answered]
Please answer, Why is it necessary? As due to it, I'm getting warning errors in my Unix-like system.

**Comment by t3chn0m4g3** [comment, marked as answer]
Have you tried to replace legacy? If it works fine without issues from glutton and honeytrap it would be great.

**Reply by t3chn0m4g3** [reply]
For T-Pot 24.x `iptables-legacy` is no longer required, updated in 234fb16394e70fae9353b6c5cb6631da7f001837

**Reply by alfredsecure** [reply]
Thanks for the update! :)

**Reply by t3chn0m4g3** [reply]
Thanks for the discussion, made me re-test it 😊

**Comment by alfredsecure** [comment]
I have replaced iptables-legacy by using this UNIX command, "sudo update-alternatives --set iptables /usr/sbin/iptables-legacy". Is this the correct way to do so?

**Reply by t3chn0m4g3** [reply]
The command you provided will set the system's default `iptables` to `iptables-legacy`. This means that when you run `iptables`, it will use `iptables-legacy` instead of the newer nftables-based `iptables`. It does not replace or remove the newer `iptables`; it just changes the symlink that determines which version of iptables is invoked by default when you run the `iptables` command.
The new `iptables` had issues in forwarding NFQ traffic to glutton and honeytrap which were solved by using `iptables-legacy`.

**Comment by alfredsecure** [comment]
Okay. How to remove this symlink, between iptables and iptables-legacy so I can see what to keep with respect to Glutton and Honeytrap behavior?
